src/SkeletalTrapezoidation.cpp

```diff
@@ -1331,27 +1194,20 @@ void SkeletalTrapezoidation::generateTransition(edge_t& edge, coord_t mid_pos, c
     float mid_rest = transition_mid_position * inner_bead_width_ratio_after_transition;
     float end_rest = inner_bead_width_ratio_after_transition;
 
-    
-        debugCheckGraphCompleteness();
-        debugCheckGraphConsistency();
-
     { // lower bead count transition end
```

Comment:
Not sure if that's for this review, but the {} aren't really needed here, right?

Reply:
No, but they're sometimes useful for documentation (and can actually help with scoping, but that isn't needed here).